packages/core/mesh/network-manager/src/transport/datachannel/index.browser.ts

```diff
@@ -0,0 +1,8 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+export const RTCPeerConnection = globalThis.RTCPeerConnection;
+export const RTCSessionDescription = globalThis.RTCSessionDescription;
+export const RTCIceCandidate = globalThis.RTCIceCandidate;
+export function cleanup(): void {}
```

Comment:
when do we call this?

Reply:
my thinking was to leave this subdir as unmodified as possible to make it easier to merge upstream changes. OTOH without proper dependency management it's effectively ours to maintain. WDYT?